definitions/divorce/json/AuthorisationCaseState/AuthorisationCaseState-nonprod.json

```diff
@@ -0,0 +1,9 @@
+[
+  {
+    "LiveFrom": "01/01/2020",
+    "CaseTypeID": "DIVORCE",
+    "CaseStateID": "DNisRefused",
+    "UserRole": "caseworker-divorce-solicitor",
+    "CRUD": "RU"
+  }
+]
```

Comment:
Added to so that generating the config files would not update the state of prod config before a deployment is required. All changes are now in lower env only